sigal/gallery.py

```diff
@@ -179,8 +179,9 @@ def __init__(self, filename, path, settings):
         base = splitext(filename)[0]
         self.date = None
         self.src_filename = filename
-        self.filename = self.url = base + '.webm'
-        self.dst_path = join(settings['destination'], path, base + '.webm')
+        if not settings['use_orig']:
+            self.filename = self.url = base + '.webm'
+            self.dst_path = join(settings['destination'], path, base + '.webm')
```

Comment:
Sorry I forgot to answer. I think that unsupported files should be converted without taking care of the `use_orig` setting, and without adding a new setting. Maybe just make it clear in the doc that `use_orig` applies only to supported files.